### PR TITLE
Fixed Spelling, Security, and Copyright Date

### PR DESCRIPTION
**Pull request by alwaysontop617**
This code should not be buggy and it should work better.

### CODE REVIEW

**Comment by alwaysontop617** [comment]
@lincanbin What do you think, its completed

**Comment by alwaysontop617** [comment]
@lincanbin Please Approve my code

File: install/index.php [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <?php
+/*
+Some Code is powered by Tecflare Corporation
+Special Thanks to Tecflare Corporation
+*/
 set_time_limit(0);
-//error_reporting(0); //don't show errors
+error_reporting(E_ERROR); //don't show errors
```
Comment (StoneMoe) [inline install/index.php]: Error reporting should be turn off completely,otherwise this will led to higher infomation leak risk

**Comment by StoneMoe** [comment]
In Open-Source world, contribute should be the first. Advertisement should not add to comment and page footer. And what's more, the code changes you made is nothing to speak of.

**Comment by alwaysontop617** [comment]
ok
I fixed it

**Comment by alwaysontop617** [comment]
@StoneMoe @lincanbin  Its all fixed, please APPROVE

**Comment by lincanbin** [comment]
Invalid.

**Pull request closed without merging**